### DISCUSSION TITLE
m2芯片的mac安装doris

### DISCUSSION THREAD

**Discussion by WYPOWLS** [apache/doris#26965] [RFC]
<img width="1404" alt="image" src="https://github.com/apache/doris/assets/45968907/36a2af06-5a20-4756-9273-64a5411251aa">
使用官网下载doris文件在本地去按照，fe启动成功了，但是启动be的时候，报如下的问题，应该怎么处理
start time: 2023年11月14日 星期二 17时31分23秒 CST
INFO: java_cmd /Users/wyp/software/jdk-17.0.9.jdk/Contents/Home/bin/java
INFO: jdk_version 17
/Users/wyp/software/doris/apache-doris-2.0.2-bin-arm64/be/lib/doris_be: /Users/wyp/software/doris/apache-doris-2.0.2-bin-arm64/be/lib/doris_be: cannot execute binary file

**Comment by WYPOWLS** [comment]
文件配置按照官网推荐使用

**Comment by zy-kkk** [comment]
To run Doris on MacOS, you need to compile the source code yourself according to the following documentation:
https://doris.apache.org/zh-CN/docs/install/source-install/compilation-mac
